app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,68 @@
+<div class="page-container">
+
+  <header>
+    <h1>Task List</h1>
+    <nav>
+      <ul class="nav-button">
```

Comment:
You should put this and the footer code in the view layout, then it will automatically be included in every page.